VERSION_5.md

```diff
@@ -0,0 +1,183 @@
+# Version 5
+
+## Introduction
+
+GEOSwift was started in May 2015 by Andrea Cremaschi. At this time Swift 1.2 had
+been out for just over a month. Over the years, the libaray and Swift both grew
+and evolved, and new features in Swift like the [error handling
+model][swift-error-handling] (Swift 2) and [Codable][swift-codable] (Swift 4)
+started to make GEOSwift feel a bit less Swifty than it did originally. Even
+early on, there was [a desire][classes-vs-structs] for GEOSwift to use structs
```

Comment:
Also think this comma is superfluous.

Reply:
I think this is okay too for the same reason.